man/import.q.concourse.Rd

```diff
@@ -2,19 +2,22 @@
 \alias{import.q.concourse}
 \title{Q methodology: import concourse of Q items}
 \description{
-  Imports full set of items (concourse) from a directory of *.tex files, including possible translations.
+  Imports full set of items (concourse) from a directory of .TEX files, including possible translations.
```

Comment:
`Imports the full set of items (the \emph{concourse}) from a directory of .TEX files, including possible translations.`